portfolio/src/main/webapp/script.js

```diff
@@ -64,14 +76,27 @@ function createCommentElement(input) {
     var commentText = input.text;
     var commentText = commentText.replace(/</g, "&lt;").replace(/>/g, "&gt;");
 
-    var commentHTML = `
-    <div class="text">${commentText}</div>
-    <div class="date">${input.postDate}</div>
-    `;
+    if(input.imageURL != null)
+    {   var commentHTML = `
```

Comment:
nit pick :D, please add curly brace to same line as if and add whitespace after if keyword

Reply:
Sure no worries, I'll fix that right away! 😄